toolkit/indoors/src/main/java/com/arcgismaps/toolkit/indoors/FloorFilter.kt

```diff
@@ -127,25 +140,103 @@ public fun FloorFilter(
                 .background(color = uiProperties.backgroundColor),
             verticalArrangement = Arrangement.Center
         ) {
+            when (initializationStatus) {
+                is InitializationStatus.NotInitialized, InitializationStatus.Initializing -> {
+                    Box(
+                        modifier = modifier
+                            .height(uiProperties.buttonSize.height.dp)
+                            .width(uiProperties.buttonSize.width.dp),
+                        contentAlignment = Center
+                    ) {
+                        CircularProgressIndicator(modifier = Modifier.size(uiProperties.buttonSize.height.dp * 0.5f))
+                    }
+                }
+
+                is InitializationStatus.FailedToInitialize -> {
+                    val errorMessage = (initializationStatus as InitializationStatus.FailedToInitialize).error.getErrorMessage(context)
+                    Log.e("FloorFilter", errorMessage)
+                    Box(
+                        modifier = modifier
+                            .height(uiProperties.buttonSize.height.dp)
+                            .width(uiProperties.buttonSize.width.dp),
+                        contentAlignment = Center
+                    ) {
+                        Icon(
+                            imageVector = Icons.Outlined.ErrorOutline,
+                            contentDescription = stringResource(id = R.string.geomodel_has_no_floor_aware_data),
+                            modifier = Modifier.clickable {
+                                Toast.makeText(context, errorMessage, Toast.LENGTH_SHORT).show()
+                            },
+                            tint = MaterialTheme.colorScheme.primary
```

Comment:
Maybe we should use the error scheme here? 
```suggestion
                            tint = MaterialTheme.colorScheme.error
```